src/components/TaskList.tsx

```
@@ -40,6 +46,9 @@ import {
   AlertDialogTitle,
 } from '@/components/ui/alert-dialog';
 
+// Lazy load the heavy AI Breakdown Modal
+const AIBreakdownModal = lazy(() => import('./AIBreakdownModal').then(module => ({ default: module.AIBreakdownModal })));
```

Biome (2.4.4)

[error] Shouldn't redeclare 'AIBreakdownModal'. Consider to delete it or rename it.

(lint/suspicious/noRedeclare)